app/js/lib/foxbox/foxbox.js

```diff
@@ -350,44 +361,6 @@ export default class Foxbox extends Service {
       });
   }
 
-  /**
-   * Fetch the state of a service from the box.
-   *
-   * @param {string} id The ID of the service.
-   * @return {Promise}
-   */
-  getServiceState(id) {
```

Comment:
I'll leave the code related to polling for now, but I'm planning on refactoring it for https://github.com/fxbox/app/pull/73.